fix: update helm chart with new values (#693)
Azure/fleet: charts/hub-agent/README.md

| namespace                       | Namespace that this Helm chart is installed on                                                                                                               | `fleet-system`                                   |
| serviceAccount.create           | Whether to create service account                                                                                                                            | `true`                                           |
| serviceAccount.name             | Service account name                                                                                                                                         | `hub-agent-sa`                                   |
| resources                       | The resource request/limits for the container image                                                                                                          | limits: 500m CPU, 1Gi, requests: 100m CPU, 128Mi |

> | affinity                        | The node affinity to use for pod scheduling                                                                                                                  | `{}`                                             |

Reviewer: ```suggestion
| affinity                        | The node affinity to use for hubagent pod                                                                                                                  | `{}`                                             |
```

> | tolerations                     | The tolerations to use for pod scheduling                                                                                                                    | `[]`                                             |

Reviewer: ```suggestion
| tolerations                     | The tolerations to use for hubagent pod                                                                                                                    | `[]`                                             |
```

> | logVerbosity                    | Log level. Uses V logs (klog)                                                                                                                                | `2`                                              |

Reviewer: ```suggestion
| logVerbosity                    | Log level. Uses V logs (klog)                                                                                                                                | `5`                                              |
```

> | enableV1Alpha1APIs              | If set, the agents will watch for the v1alpha1 APIs.                                                                                                         | `true`                                           |

Reviewer: default should be false

> | enableV1Beta1APIs               | If set, the agents will watch for the v1beta1 APIs.                                                                                                          | `false`                                          |

Reviewer: this should be false

| hubAPIQPS                       | QPS to use while talking with fleet-apiserver. Doesn't cover events and node heartbeat apis which rate limiting is controlled by a different set of flags.   | `20.0`                                           |

> | hubAPIBurst                     | Burst to use while talking with fleet-apiserver. Doesn't cover events and node heartbeat apis which rate limiting is controlled by a different set of flags. | `10`                                             |

Reviewer: burst should be higher than QPS, probably set to 100

> | concurrentClusterPlacementSyncs | The number of cluster placement reconcilers to run concurrently.                                                                                             | `10`                                             |

Reviewer: ```suggestion
| concurrentClusterPlacementSyncs | The number of clusterResourcePlacement to run concurrently.                                                                                             | `10`                                             |
```

> | concurrentClusterPlacementSyncs | The number of clusterResourcePlacement to run concurrently.                                                                                                  | `10`                                             |

Reviewer: ```suggestion
| concurrentClusterPlacementSyncs | The max number of clusterResourcePlacement to run concurrently this fleet supports.                                                                                                  | `10`                                             |
```

Reviewer: given the name is not what it looks like, can we change the name?

Reviewer: do you want to change this in a following up PR? I mean change the "concurrentClusterPlacementSyncs" to reflect what it actually it

Author: I'll change it on here.

| ConcurrentResourceChangeSyncs   | The number of resourceChange reconcilers that are allowed to run concurrently.                                                                               | `20`                                             |

> | logFileMaxSize                  | Max size of log file                                                                                                                                         | `1800`                                           |

Reviewer: why is this 1800 instead of 1000000?

> | logFileMaxSize                  | Max size of log file                                                                                                                                         | `1000000`                                        |

Reviewer: ```suggestion
| logFileMaxSize                  | Max size of log file before rotation                                                                                                                                       | `1000000`                                        |
```
